R/sits_tibble_functions.R

```diff
@@ -209,15 +209,15 @@ sits_dates <- function(data.tb) {
 #' @name sits_merge
 #' @author Gilberto Camara, \email{gilberto.camara@@inpe.br}
 #'
-#' @description This function merges the time series of two STIS tables.
+#' @description This function merges the time series of two STIS tibbles.
```

Comment:
Typo
```suggestion
#' @description This function merges the time series of two SITS tibbles.
```

Reply:
I fixed this in a separate commit as it is better to write using lowercase.